databrowse/plugins/db_datacollect_v2_viewer/db_datacollect_v2_viewer.py

```diff
@@ -414,7 +424,7 @@ def loadStyleFunction(self):
 
         #print "Stylesheet Loaded Successfully:"
         #print stylestring
-        stylestring = stylestring.replace('/usr/local/QAutils/checklist/datacollect2.xsl', os.path.join(self._web_support.qautils, "checklist/datacollect2.xsl"))
+        stylestring = stylestring.replace('/usr/local/QAutils/checklist/datacollect2.xsl', "/".join([self._web_support.qautils, "checklist/datacollect2.xsl"]))
```

Comment:
I'm a bit curious as to why this was changed.  I would expect using os.path to be more cross platform compliant.